### ISSUE TITLE
Installed - nothing happens?

### ISSUE THREAD

**Issue by copernicus365** [AdamAnandUS/AdamsTool#1]
That's it: I installed it, and it says it's installed in extensions, but no sign of a toolbar anywhere. Am I missing a boolean somewhere?

**Comment by AdamAnandUS** [comment]
You should see toolbar with buttons in the bottom status bar. Which version of Visual Studio code you are using?

Cheers
Adam 
 
  On Thu, Oct 25, 2018 at 2:06 PM, Nicholas Petersen<notifications@github.com> wrote:   
That's it: I installed it, and it says it's installed in extensions, but no sign of a toolbar anywhere. Am I missing a boolean somewhere?

**Comment by copernicus365** [comment]
Oh, I see it now, it is with the status bar on the bottom. Thanks!

**Comment by AdamAnandUS** [comment]
Perfect. Thank you

**Comment by doniks** [comment]
just came here as well, because I couldn't see it :)

Maybe rephrase the readme somehow? #3
